compose/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+
+services:
+  letsencrypt:
+    image: adferrand/letsencrypt-dns
+    container_name: "letsencrypt-dns"
+    env_file: .env
+    volumes:
+      - "$ROOT/domains.conf:/etc/letsencrypt/domains.conf"
+      - "$ROOT/certs/:/etc/letsencrypt"
+    environment:
```

Comment:
Do we need to change the name of the variables in `.env` thus requiring this section?

👍 from me other than that.

Reply:
Indeed, one would expect that the variables defined in `.env` would correspond to the one used in the `docker-compose.yml` file.

Reply:
@adferrand the purpose of the version is the  docker-compose.yml version. Detailed information at https://docs.docker.com/compose/compose-file/compose-versioning/

Yes the variables in the .env file relate to ones used in the docker-compose file.
Explainaiton can be found at: https://docs.docker.com/compose/environment-variables/

Reply:
I meant the `-VERSION=latest` line in the `environment` defined for the docker.

Otherwise you are right, all variables are exposed correctly.

Reply:
That is just that you are able to select the version from hub.docker.com or wherever you deployed it. Just to add this level of control also within the compose configuration.

Reply:
Ok, great !